debezium-core/src/main/java/io/debezium/util/Clock.java

```diff
@@ -37,14 +38,17 @@ static Clock system() {
         return SYSTEM;
     }
 
+    default Instant currentTime() {
+        return Instant.ofEpochMilli(currentTimeInMillis());
```

Comment:
Any reason to not use `Instant.now()`? Is that to allow for mocking the Clock in tests?

Reply:
Yes, the idea is to make `Clock` implementations pluggable (and their only required method is `currentTimeInMillis()`.